---
ClampArray clamps
int32[] model_ids
